products/numpy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NumPy
+permalink: /numpy
+category: framework
+releasePolicyLink: https://numpy.org/neps/nep-0029-deprecation_policy.html
+versionCommand: python -c "import numpy; print(numpy.__version__)"
+auto:
+-   git: https://github.com/numpy/numpy
+-   regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)$
+releases:
+-   releaseCycle: "1.14"
+    eol: 2020-01-07
+    releaseDate: 2018-01-06
+
+-   releaseCycle: "1.15"
+    eol: 2020-07-23
+    releaseDate: 2018-07-23
+
+-   releaseCycle: "1.16"
+    eol: 2021-01-13
+    releaseDate: 2019-01-13
+    
+-   releaseCycle: "1.17"
+    eol: 2021-07-26
+    releaseDate: 2019-07-26
+    
+-   releaseCycle: "1.18"
+    eol: 2021-12-22
+    releaseDate: 2019-12-22
+    
+-   releaseCycle: "1.19"
+    eol: 2022-06-21
+    releaseDate: 2020-06-20
+    
+-   releaseCycle: "1.20"
+    eol: 2023-01-31
+    releaseDate: 2021-01-31
+    
+-   releaseCycle: "1.21"
+    eol: 2023-06-23
+    releaseDate: 2021-06-22
+    
+-   releaseCycle: "1.22"
+    eol: 2024-01-01
+    releaseDate: 2021-12-31
+  
+---
+  
+> [NumPy](https://numpy.org/) offers comprehensive mathematical functions, random number generators, linear algebra routines, Fourier transforms, and more.
```

Comment:
Add a few lines summarizing NEP29 below this please. Important things to cover are:

- Release Cadence
- Versioning Scheme (Semver etc)
- Support policy (24 months, and last-3 at minimum)

A clear mention of NEP-29 applying to various other projects will also be helpful.

Reply:
I've given it a shot, but I'm not sure if my sentence summarizes it appropriately / if I actually understood their policy. I'm not aware of other projects that use numpys deprecation scheme, but I've left a direct link to the document in the summary.

From the historic dates, it looks like they are releasing roughly every 6 months, but I didn't find a document stating that explicitly. Do you think we should still mention that they release twice a year?

Reply:
Regarding semver, the document just simply states:

> For the purposes of this NEP we assume semantic versioning and define:

But otherwise, their versioning also looks to me like semver, so I think it's safe to assume that they are actually doing semver.

Reply:
Worth mentioning with a "usually" caveat. We can raise a request to NumPy to review once we have a first version.